### DISCUSSION TITLE
Hacktober Fest

### DISCUSSION THREAD

**Discussion by thesujai** [learningequality/kolibri#11356] [Q&A] [answered]
Will this repository participate in `Hacktober Fest`?

**Comment by MisRob** [comment]
Hello @thesujai, I will reach out back in a few days in regards to Hactoberfest

**Comment by MisRob** [comment, marked as answer]
@thesujai Even though we won't participate in Hactoberfest actively this year, we can put `hacktoberfest-accepted` label to your pull requests as you mentioned in https://github.com/learningequality/kolibri/issues/11321#issuecomment-1751983790.

I think some of your work has already been labeled.

Please check all your pull requests and if there are some more we merged but don't have the label, send me links here and I will label the remaining ones. This also applies to your future work - I would ask you to remind it to us in a PR.

**Reply by thesujai** [reply]
Thanks @MisRob, yes all of PRs are Labelled. Thank you
